### PR TITLE
Backport of #177 and tagging  from `Manager-5.0-Beta1` to `main`

### PR DESCRIPTION
**Pull request by deneb-alpha**
## What does this PR change?

Backport of #177 and tagging  from `Manager-5.0-Beta1` to `main`

## Test coverage
- No tests: **just spec file changes**

- [x] **DONE**

## Links

Relate(s): https://github.com/SUSE/spacewalk/issues/23806, https://github.com/SUSE/spacewalk/issues/23610

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by raulillo82 — APPROVED** [review]
LGTM

**Comment by deneb-alpha** [comment]
of course,  the failure  on the changelog test  is  expected when we  backport  the  tagging.

**Review by cbosdo — APPROVED** [review]

**Comment by deneb-alpha** [comment]
@juliogonzalez BTW, as requested  by autobuild, for tagging I used `tito tag --use-release=0` for having  `Release: 0` instead of the default `Release: 1`

File: uyuni-tools.spec [hunk 1]
```diff
@@ -43,8 +43,8 @@
 %endif
 
 Name:           %{project}
-Version:        0.1.4
-Release:        1
+Version:        0.1.5
+Release:        0
```
Comment (juliogonzalez) [inline uyuni-tools.spec:47]: I think autobuild requested `0`, right? (If so, this is correct)

**Review by juliogonzalez — APPROVED** [review]
LGTM, just one comment, as I am not sure.

File: uyuni-tools.spec (hunk 1 above)
Comment (deneb-alpha) [reply uyuni-tools.spec:47]: yes, see https://github.com/uyuni-project/uyuni-tools/pull/179#issuecomment-1966651801

Comment (juliogonzalez) [reply uyuni-tools.spec:47]: Argh, I am blind :facepalm:

**Pull request merged**